webCommander/centralHub/views.py

```diff
@@ -36,3 +37,17 @@ def handle_summary(request, twitter_handle):
 
 def home_page(request):
     return render(request, 'home.html')
+
+def list_artist(request):
+    allartists = SpotifyArtistInfo.objects.all()
+
+    return render(request, 'spotifyFramework/spotify_summary.html',{'allartists':allartists})
+
+def get_artists(request):
```

Comment:
do we want to add @login_required to make it only for logged in users? so nobody goes DOS attack by making many requests to the Spotify API exhausting the API rate limits?